runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowSystemMetrics.java

```diff
@@ -37,7 +37,8 @@
     WINDMILl_STATE_BYTES_WRITTEN("WindmillStateBytesWritten"),
     JAVA_HARNESS_USED_MEMORY("dataflow_java_harness_used_memory"),
     JAVA_HARNESS_MAX_MEMORY("dataflow_java_harness_max_memory"),
-    JAVA_HARNESS_RESTARTS("dataflow_java_harness_restarts");
+    JAVA_HARNESS_RESTARTS("dataflow_java_harness_restarts"),
+    WINDMILL_QUOTA_THROTTLING("dataflow_throttled_msecs");
```

Comment:
It seems better to treat the global throttling signal separate from the per-stage ones, on the service-side, so I suggest a different name, perhaps: dataflow_streaming_engine_throttled_msecs?

Reply:
Done